src/IQFeed.CSharpApiClient/Lookup/Symbol/Messages/ListedMarketMessage.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Globalization;
+using IQFeed.CSharpApiClient.Extensions;
+
+namespace IQFeed.CSharpApiClient.Lookup.Symbol.Messages
+{
+    public class ListedMarketMessage : IListedMarket
```

Comment:
any reason to add Interface on immutable types?

Reply:
I was just following the existing pattern. The only valid reason I could think of to have a read only clean interface without RequestId. If it was for me, I would not have them

Reply:
removed